src/dlgprefshoutcast.cpp

```diff
@@ -257,7 +293,35 @@ void DlgPrefShoutcast::slotApply()
     m_pConfig->set(ConfigKey(SHOUTCAST_PREF_KEY, "custom_artist"), ConfigValue(custom_artist->text()));
     m_pConfig->set(ConfigKey(SHOUTCAST_PREF_KEY, "custom_title"),  ConfigValue(custom_title->text()));
     m_pConfig->set(ConfigKey(SHOUTCAST_PREF_KEY, "metadata_format"), ConfigValue(metadata_format->text()));
+}
+
+void DlgPrefShoutcast::setDialogEnabled(bool enabled) {
+        comboBoxServerType->setEnabled(enabled);
```

Comment:
it's ok, you don't need to come up with a reason for what you did :).  a simple "done" is fine enough and then you don't have to spend time thinking about what to say.